### PR TITLE
Sergiohdljr feat/front/add react query onsub calls

### PR DESCRIPTION
**Pull request by anti-duhring**

### CODE REVIEW

File: apps/web/app/landing-page/Hero.tsx [hunk 1]
```diff
@@ -38,16 +40,15 @@ export const Hero = () => {
...
-  }, [])
+  const getSubscribersCount = async () => {
+    const response = await fetch(ApiRoutes.Subscribers)
+    if (response.ok) {
+      const count = await response.json()
+      return count
+    }
+  }
 
+  const { data:subscribersCount } = useQuery<number>( 'subscribres', async () => await getSubscribersCount())
```
Comment (ocodista) [inline apps/web/app/landing-page/Hero.tsx:51]: The lint seems broken, there shouldn't be a space before 'subscribers'.

It has also a typo.

@anti-duhring haven't you merged the previous PR into this one?

I kinda preferred the other name. (with count)

Comment (ocodista) [inline apps/web/app/landing-page/Hero.tsx:51]:
```suggestion
  const { data: subscribersCount } = useQuery<number>('subscribersCountQuery', async () => await getSubscribersCount())
```

**Review by ocodista — CHANGES_REQUESTED** [review]
@anti-duhring was there any merge errors when merging the previous PR with this one?

**Comment by anti-duhring** [comment]
> @anti-duhring was there any merge errors when merging the previous PR with this one?

I couldn't resolve the `yarn.lock` conflict in that branch, so I cloned that PR into this branch

File: apps/web/app/landing-page/Hero.tsx [hunk 2]
```diff
@@ -38,16 +40,15 @@ export const Hero = () => {
...
-      }
-    })()
-  }, [])
+  const getSubscribersCount = async () => {
+    const response = await fetch(ApiRoutes.Subscribers)
+    if (response.ok) {
+      const count = await response.json()
+      return count
+    }
+  }
```
Comment (ocodista) [inline apps/web/app/landing-page/Hero.tsx:49]: Please add type and return something.

```suggestion
  const getSubscribersCount = async (): Promise<number | null> => {
    const response = await fetch(ApiRoutes.Subscribers)
    if (!response?.ok) return null
    const count = await response.json()
    return count
  }
```

**Review by ocodista — CHANGES_REQUESTED** [review]

**Review by ocodista — APPROVED** [review]

**Pull request merged**